testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/OfflineTokenTest.java

```diff
@@ -944,12 +948,64 @@ private void offlineTokenRequest(String expectedRefreshAlg, String expectedAcces
                 .assertEvent();
 
         // Refresh with both offline tokens is fine
-        testRefreshWithOfflineToken(token, offlineToken, offlineTokenString, token.getSessionState(), serviceAccountUserId);
-        testRefreshWithOfflineToken(token2, offlineToken2, offlineTokenString2, token2.getSessionState(), serviceAccountUserId);
+        testRefreshWithOfflineToken(token, offlineToken, offlineTokenString, token.getSessionState(), serviceAccountUserId, false);
+        testRefreshWithOfflineToken(token2, offlineToken2, offlineTokenString2, token2.getSessionState(), serviceAccountUserId, false);
 
     }
 
-    @Test
+    private void offlineTokenRequestWithScopeParameter(String expectedRefreshAlg, String expectedAccessAlg, String expectedIdTokenAlg) throws Exception {
```

Comment:
This method is not used anywhere. Could it be deleted from this PR? IMO the test coverage is sufficient by other methods of this PR (like in RefreshTokenTest). This means that new argument `scopeParameter` can be removed from the method `refreshAccessToken` and just assume that `offline_token` scope is always present in access token? Will simplify a lot changes needed in the `OfflineTokenTest` class . Probably almost not changes would be needed in the class - besides checking that `refreshedToken` and `newRefreshTokenFull` still contains `offline_access` scope.

Reply:
You are right. It will be removed.